.github/workflows/unit-tests.yml

```diff
@@ -17,22 +17,12 @@ jobs:
           - "3.8"
         django-version:
           - "pinned"
-          #- "4.0"
+          - "4.0"
         shard_name:
           - "lms-1"
-          - "lms-2"
-          - "lms-3"
-          - "lms-4"
-          - "lms-5"
-          - "lms-6"
           - "openedx-1"
-          - "openedx-2"
-          - "openedx-3"
-          - "openedx-4"
           - "cms-1"
-          - "cms-2"
           - "common-1"
-          - "common-2"
```

Comment:
The deletions here and the addition of 4.0 to the matrix are for testing purposes only and will need to be backed out before merge. (Adding 4.0 tests that it fails appropriately, and removing shards helps keep the runtime down.)